nestkernel/archiving_node.h

```diff
@@ -50,44 +50,32 @@ class ArchivingNode : public StructuralPlasticityNode
 {
 public:
   /**
-   * \fn ArchivingNode()
-   * Constructor.
+   * Constructor
    */
```

Comment:
Trivial comments like this on constructor, copy constuctor or assignment operator provide no information. I would remove them completely.

Reply:
Removed these, also removed destructor comments as well